components/user_settings/display/user_settings_theme/user_settings_theme.jsx

```diff
@@ -86,23 +86,24 @@ export default class ThemeSetting extends React.Component {
         $('.ps-container.modal-body').scrollTop(0);
     }
 
-    submitTheme = () => {
+    submitTheme = async () => {
```

Comment:
There are some functions that are executed after `savePreferences`, so instead of using Promise chain I opted to use `async/await`, and plus it is used in tests as an indication of function completion to start checking if some conditions satisfy